build_tools/kickstarts/centos-7-adb-vagrant.ks

```diff
@@ -70,6 +70,12 @@ yum install -y docker-selinux
 # set tuned profile to force virtual-guest
 tuned-adm profile virtual-guest
 
+OPENSHIFT_TAG="v1.1.6"
+
+# Edit sccli.py and openshift_options as per openshift_version
```

Comment:
This comment needs to be updated.  Suggested:

Edit openshift_options to reflect the desired default version of OpenShift